### PR TITLE
Layout and Show detail book

### PR DESCRIPTION
**Pull request by mvn-nale-dn**
## Description
Create layout and show detail book
## Reference Requirements
- Task ID: #63

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)
```
explain select * from `books` where `books`.`id` = '1' and `books`.`deleted_at` is null limit 1\G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: books
   partitions: NULL
         type: const
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: const
         rows: 1
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.02 sec)

```
```
explain select * from `categories` where `categories`.`id` in ('8')\G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: categories
   partitions: NULL
         type: const
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: const
         rows: 1
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.00 sec)
```
```
explain select * from `image_books` where `image_books`.`book_id` in ('1')\G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: image_books
   partitions: NULL
         type: ref
possible_keys: image_books_book_id_foreign
          key: image_books_book_id_foreign
      key_len: 4
          ref: const
         rows: 1
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.00 sec)
```
- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by mvn-haphan-dn — COMMENTED** [review]
LGTM

File: app/Http/Controllers/Admin/BookController.php [hunk 1]
```diff
@@ -148,4 +148,17 @@ public function destroy(Book $book)
...
     }
+
+     /**
+     * Display the detail of book.
+     *
+     * @param int $id id of book
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function show ($id)
```
Comment (at-haphan) [inline app/Http/Controllers/Admin/BookController.php]: show($id)

**Review by mvn-haphan-dn — COMMENTED** [review]
LGTM

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**